.circleci/deploy-python-2.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+
+set -e
+
+VERSION_CHANGE_TRIGGERS="setup.py MANIFEST.in openfisca_core openfisca_web_api_preview"
+
+if git diff-index --quiet HEAD^ -- $VERSION_CHANGE_TRIGGERS ":(exclude)*.md"
```

Comment:
We're only checking for changes in the last commit, not in the whole branch. Is that expected?

Reply:
There is a subtlety here.

We are in the context of deployment. So there is not more feature branch involved: we are on `master`, and want to know if the new version of master brings any functional change compared to the last version of master. Hence the comparison with `HEAD^`. 

This is different when we are testing a feature branch, and running `check-version-bump.sh`. In that case, we indeed want to compare our branch to the current version of master.

So we can't really share the whole `detect-functional-changes.sh` as is. I'll try to find a solution though.